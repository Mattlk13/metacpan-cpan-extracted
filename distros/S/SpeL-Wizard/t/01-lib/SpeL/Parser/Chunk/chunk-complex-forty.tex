\(\ds \left (\frac {1+\jz }{1-\jz }\right )^{40}\)
