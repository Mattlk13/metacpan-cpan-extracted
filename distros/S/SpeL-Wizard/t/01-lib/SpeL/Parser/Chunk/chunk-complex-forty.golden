$tex = bless( {
  'path' => 't/01-lib/SpeL/Parser/Chunk',
  'tree' => bless( {
    'ElementList' => bless( {
      'Element' => [
        bless( {
          'MathUnit' => bless( {
            '' => '\\ds \\left (\\frac {1+\\jz }{1-\\jz }\\right )^{40}',
            'Left' => bless( {
              'Component' => bless( {
                'Name' => 'ds',
                'trailingws' => ' '
              }, 'SpeL::Object::Command' ),
              'Rest' => bless( {
                'Op' => bless( {
                  'Op' => ''
                }, 'SpeL::Object::Binop' ),
                'Remainder' => bless( {
                  'Component' => bless( {
                    'Component' => bless( {
                      '' => '\\frac {1+\\jz }{1-\\jz }',
                      'Left' => bless( {
                        'Component' => bless( {
                          'den' => bless( {
                            'Component' => bless( {
                              'Realnumber' => bless( {
                                'Sign' => '',
                                'Value' => '1',
                                'trailingws' => ''
                              }, 'SpeL::Object::Realnumber' )
                            }, 'SpeL::Object::Number' ),
                            'Rest' => bless( {
                              'Op' => bless( {
                                'Op' => '-'
                              }, 'SpeL::Object::Binop' ),
                              'Remainder' => bless( {
                                'Component' => bless( {
                                  'Name' => 'jz',
                                  'trailingws' => ' '
                                }, 'SpeL::Object::Command' ),
                                'Rest' => bless( {
                                  'ws' => ''
                                }, 'SpeL::Object::Expressionrest' )
                              }, 'SpeL::Object::Expression' )
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' ),
                          'num' => bless( {
                            'Component' => bless( {
                              'Realnumber' => bless( {
                                'Sign' => '',
                                'Value' => '1',
                                'trailingws' => ''
                              }, 'SpeL::Object::Realnumber' )
                            }, 'SpeL::Object::Number' ),
                            'Rest' => bless( {
                              'Op' => bless( {
                                'Op' => '+'
                              }, 'SpeL::Object::Binop' ),
                              'Remainder' => bless( {
                                'Component' => bless( {
                                  'Name' => 'jz',
                                  'trailingws' => ' '
                                }, 'SpeL::Object::Command' ),
                                'Rest' => bless( {
                                  'ws' => ''
                                }, 'SpeL::Object::Expressionrest' )
                              }, 'SpeL::Object::Expression' )
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' )
                        }, 'SpeL::Object::Fraction' ),
                        'Rest' => bless( {
                          'ws' => ''
                        }, 'SpeL::Object::Expressionrest' )
                      }, 'SpeL::Object::Expression' )
                    }, 'SpeL::Object::Relation' ),
                    'cbracket' => '\\right )',
                    'obracket' => '\\left ('
                  }, 'SpeL::Object::Bracketconstruction' ),
                  'Rest' => bless( {
                    'Op' => bless( {
                      'Group' => bless( {
                        'MathUnit' => bless( {
                          '' => '40',
                          'Left' => bless( {
                            'Component' => bless( {
                              'Realnumber' => bless( {
                                'Sign' => '',
                                'Value' => '40',
                                'trailingws' => ''
                              }, 'SpeL::Object::Realnumber' )
                            }, 'SpeL::Object::Number' ),
                            'Rest' => bless( {
                              'ws' => ''
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' )
                        }, 'SpeL::Object::Relation' ),
                        'trailingws' => ''
                      }, 'SpeL::Object::MathGroup' ),
                      'sup' => '^'
                    }, 'SpeL::Object::Power' ),
                    'Remainder' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expressionrest' )
                }, 'SpeL::Object::Expression' )
              }, 'SpeL::Object::Expressionrest' )
            }, 'SpeL::Object::Expression' )
          }, 'SpeL::Object::Relation' )
        }, 'SpeL::Object::MathInline' )
      ]
    }, 'SpeL::Object::ElementList' )
  }, 'SpeL::Object::Document' )
}, 'SpeL::Parser::Chunk' );

